CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows conventions [outlined here](http://keepachangelog.com/).
 
+
+## vNEXT 
```

Comment:
We should always do this

Reply:
yeah, but remembering to do it is :confounded: